tools/python/boutiques/tests/test_publisher.py

```diff
@@ -6,9 +6,16 @@
 
 class TestPublisher(TestCase):
 
-    def get_boutiques_dir(self):
-        return os.path.join(os.path.split(bfile)[0], "..", "..", "..")
+    def get_examples_dir(self):
+        return os.path.join(os.path.dirname(bfile),
+                            "schema", "examples")
 
     def test_publisher(self):
-        self.assertFalse(bosh(["publish", self.get_boutiques_dir(),
-                               "test author", "example.com", "--no-github"]))
+        example1_dir = os.path.join(self.get_examples_dir(), "example1")
+        print(os.path.join(example1_dir, "example1.json"))
+        self.assertFalse(bosh(["publish",
+                               os.path.join(example1_dir, "example1.json"),
+                               "Test author", "Test affiliation",
+                               "--sandbox", "-y", "-v",
+                               "--zenodo-token", "hAaW2wSBZMskxpfigTYHcuDrC"
+                               "PWr2VeQZgBLErKbfF5RdrKhzzJi8i2hnN8r"]))
```

Comment:
Right after this, in the same test definition (to ensure the above was run first), I added the following to test the token-reading-from-file functionality of `bosh publish`:

```
self.assertFalse(bosh(["publish",
                            os.path.join(example1_dir, "example1.json"),
                            "Test author", "Test affiliation",
                            "--sandbox", "-y", "-v"]))
```